psyneulink/library/compositions/autodiffcomposition.py

```diff
@@ -563,6 +563,27 @@ def execute(self,
                                                         report_num=report_num
                                                         )
 
+    @handle_external_context()
+    def save(self, path:str, context=None):
+        """Saves all parameters to the specified path (e.g. save('my_model.pt'))"""
+        assert (str, "Must provide a path to save the model to!")
```

Comment:
This is not a correct way to write assertions, both linter and python should complain about it